src/cmd/ship.go

```diff
@@ -128,7 +133,10 @@ func gitShipConfig(args []string, driver drivers.CodeHostingDriver, repo *git.Pr
 	if err != nil {
 		return result, err
 	}
-	result.hasOrigin = git.HasRemote("origin")
+	result.hasOrigin, err = repo.Silent.HasRemote("origin")
```

Comment:
Same here as `repo.Silent.HasRemote("origin")` is called twice in this function.

Reply:
Confirmed